examples/tests/multi_alarm_simple_test/main.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <libtock-sync/services/alarm.h>
+
+static void event_cb(__attribute__ ((unused)) uint32_t now,
+                     __attribute__ ((unused)) uint32_t expiration,
+                     void*                             ud) {
```

Comment:
```suggestion
static void event_cb(uint32_t now, uint32_t expiration, void* ud) {
```

I'm mildly annoyed that using an "unused"isn't some kind of warning...